src/platforms/common/performance/instrumentation/opentelemetry.mdx

```diff
@@ -33,12 +32,94 @@ description: "Using OpenTelemetry with Sentry Performance."
 
 <Include name="beta-note-opentelemetry.mdx" />
 
+<PlatformSection notSupported={["java"]}>
+
 You can configure your [OpenTelemetry SDK](https://opentelemetry.io/) to send traces and spans to Sentry.
 
+</PlatformSection>
+
+<PlatformSection supported={["java"]}>
+
+There are multiple ways of configuring your [OpenTelemetry SDK](https://opentelemetry.io/) to send traces and spans to Sentry.
+
+</PlatformSection>
+
+## Preface
```

Comment:
Please helps improve this section to make it accurate and easy to understand.

Reply:
<img width="810" alt="image" src="https://user-images.githubusercontent.com/363802/216025025-4ee328c3-535b-4196-abd3-212680efc091.png">

```
An OpenTelemetry Span can become a Sentry Transaction or Span. The first Span sent through the Sentry SpanProcessor is a Transaction, and any child Span of it is attached to the Transaction by checking the parent Span context. This happens for the OpenTelemetry root Span and any top-most Span in the system. For example, a request from frontend to backend creates an OpenTelemetry root Span and its corresponding Sentry Transaction. The backend request creates a new Sentry Transaction for its OpenTelemetry Span. The Sentry Transaction and Span are linked as a trace for navigation and error tracking.
```